GOVERNANCE.md

```diff
@@ -0,0 +1,54 @@
+# Kiali Governance
+
+This document defines governance policies for the Kiali project.
+
+## Maintainers
+
+Kiali Maintainers have write access to the Kiali GitHub repositories https://github.com/kiali.
+They can merge their own patches or patches from others. The current maintainers can be found in https://github.com/orgs/kiali/teams/maintainers.
+
+This privilege is granted with some expectation of responsibility: maintainers are people who care about the Kiali project and want to help it grow and improve. A maintainer is not just someone who can make changes, but someone who has demonstrated his or her ability to collaborate with the team, get the most knowledgeable people to review code, contribute high-quality code, and follow through to fix issues (in code or tests).
+
+A maintainer is a contributor to the Kiali project's success and a citizen helping the project succeed.
+
+## Becoming a Maintainer
+
+To become a maintainer you need to demonstrate the following:
+
+  * commitment to the project
+    * participate in discussions, contributions, code reviews for 3 months or more,
+    * perform code reviews for 10 non-trivial pull requests,
+    * contribute 10 non-trivial pull requests and have them merged into master,
+  * ability to write good code,
+  * ability to collaborate with the team,
+  * understanding of how the team works (policies, processes for testing and code review, etc),
+  * understanding of the projects' code base and coding style.
+
+A new maintainer must be proposed by an existing maintainer by sending a message to the
+[kiali-dev@googlegroups.com](https://groups.google.com/forum/#!forum/kiali-dev)
+mailing list containing the following information:
+
+  * nominee's first and last name,
+  * nominee's email address and GitHub user name,
+  * an explanation of why the nominee should be a committer,
+  * a list of links to non-trivial pull requests (top 10) authored by the nominee.
+
+Two other maintainers need to second the nomination. If no one objects in 5 working days (U.S.), the nomination is accepted.  If anyone objects or wants more information, the maintainers discuss and usually come to a consensus (within the 5 working days). If issues can't be resolved, there's a simple majority vote among current maintainers.
+
+## Changes in Maintainership
+
+Maintainers can be removed by a 2/3 majority vote by maintainers.
```

Comment:
Question: when the maintainer agrees to leave (e.g. doesn't have time for the job) or is self-resigning, perhaps we don't have to go through a vote?

Reply:
To me that situation may fall on this case as well.
There is some maintainer that would like to drop for whatever reason, then the group comment on that before.

Note, in 99% of the cases these votes don't need to be "formal", just when proposing some action, if someone desagrees and there is no unanimous decision, then it can be setup a formal vote.

I think.

Reply:
The way it's written here, it sounds very formal. If we don't like the way it's written, then we should rewrite. In my (very small) experience of OSS with open governance, votes are systematic, even if it always ends up in an unanimous "+1".

Reply:
I don't understand.
There will be a vote.
Another topic is how that vote is.

An email can work as a vote, or a thumbup or a differente mechanism, this doc shouldn't force about the how, but about the what.

What I understand is that there is a no consensus of that minimum amount of maintainers, an actions shouldn't go there.

How we setup the votes is a different discussion, in my opinion.

Reply:
We didn't have the case for a maintainer being removed yet, but I would expect a vote to happen, even if just a symbolic one.

Reply:
@jpkrohling yes I think that's the "spirit" of this kind of governance.
Though for Kiali this is a bit weird given the number of maintainers being all from the same team (at least at the moment)

Reply:
+1, "Maintainers can be removed by a 2/3 majority vote by maintainers, or by resignation."